README.md

```diff
@@ -1,6 +1,6 @@
 # Kaspa on Rust
 
-Work in progress to implement the Kaspa full-node and related libraries in the Rust programming language.
+An alpha version for a Kaspa full-node and related libraries in the Rust programming language.

```

Comment:
I think `for a` -> `for the` sounds more correct.

Or maybe:

This repository contains the implementation of the Kaspa full-node and related libraries in the Rust programming language. This is an Alpha version at the initial testing phase, however the node is expected to be fully functional and capable as a drop-in replacement for the Kaspa golang node.